code/game/objects/items/manuals.dm

```diff
@@ -26,15 +26,17 @@
 				<body>
 				<h3>Growing Humans</h3>
 
-				Why would you want to grow humans? Well I'm expecting most readers to be in the slave trade, but a few might actually
-				want to revive fallen comrades. Growing pod people is easy, but prone to disaster.
+				Why would you want to grow humans? Well, I'm expecting most readers to be in the slave trade, but a few might actually
+				want to revive fallen comrades. Growing pod people is actually quite simple:
 				<p>
 				<ol>
-				<li>Find a dead person who is in need of cloning. </li>
-				<li>Take a blood sample with a syringe. </li>
-				<li>Inject a seed pack with the blood sample. </li>
+				<li>Find a dead person who is in need of revival. </li>
+				<li>Take a blood sample with a syringe (samples of their blood taken BEFORE they died will also work). </li>
+				<li>Inject a packet of replica pod seeds (which can be acquired by either mutating cabbages into replica pods (and then harvesting said replica pods) or by purchasing them from certain corporate entities) with the blood sample. </li>

```

Comment:
The brackets inside the brackets looks weird.

Reply:
Yeah, but I don't know where else to put that (important) information.